tidb-distributed-execution-framework.md

```diff
@@ -27,29 +27,29 @@ Enabling the DXF can solve the above problems and has the following three advant
 - The DXF supports distributed execution of tasks, which can flexibly schedule the available computing resources of the entire TiDB cluster, thereby better utilizing the computing resources in a TiDB cluster.
 - The DXF provides unified resource usage and management capabilities for both overall and individual tasks.
 
-Currently, the DXF supports the distributed execution of the `ADD INDEX` and `IMPORT INTO` statements.
+Currently, the DXF supports the distributed execution of the [`ADD INDEX`] and [`IMPORT INTO`] statements.
 
-- `ADD INDEX` is a DDL statement used to create indexes. For example:
+- [`ADD INDEX`] is a DDL statement used to create indexes. For example:
 
     ```sql
     ALTER TABLE t1 ADD INDEX idx1(c1);
     CREATE INDEX idx1 ON table t1(c1);
     ```
 
-- `IMPORT INTO` is used to import data in formats such as `CSV`, `SQL`, and `PARQUET` into an empty table. For more information, see [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md).
+- [`IMPORT INTO`] is used to import data in formats such as CSV, SQL, and Parquet into an empty table.
 
 ## Limitation
 
-The DXF can only schedule up to 16 tasks (including `ADD INDEX` tasks and `IMPORT INTO` tasks) simultaneously. 
+The DXF can only schedule up to 16 tasks (including [`ADD INDEX`] tasks and [`IMPORT INTO`] tasks) simultaneously. 
 
 ## `ADD INDEX` limitation
 
-- For each cluster, only one `ADD INDEX` task is allowed for distributed execution at a time. If a new `ADD INDEX` task is submitted before the current `ADD INDEX` distributed task has finished, the new `ADD INDEX` task is executed through a transaction instead of being scheduled by DXF.
+- For each cluster, only one [`ADD INDEX`] task is allowed for distributed execution at a time. If a new [`ADD INDEX`] task is submitted before the current [`ADD INDEX`] distributed task has finished, the new [`ADD INDEX`] task is executed through a transaction instead of being scheduled by DXF.
```

Comment:
Since 8.1, multiple add index jobs can be run in parallel by DXF @hfxsd

Reply:
Maybe we can update this line after merging this PR and then only apply it to v8.1 and master? Or should we change it here and then make sure to not apply this to release-8.0?

Reply:
Fixed here https://github.com/pingcap/docs/pull/17637/files PTAL @Frank945946 @dveeden